plugins/foreman_remote_execution/0.1/index.md

```diff
@@ -0,0 +1,394 @@
+find writing job templates familiar. First navigate to Host > Job
+Templates (new menu entry). Then hit the green button with label "New
+Job Template".
+
+Note: If you don't see menu entry, the Foreman plugin is not installed
+correctly.
+
+Now you should see form similar to Provisioning template one. In first
+tab you have to fill in **Name**. You can use input macros in ERB tags
+which stands for variables which you define in second tab called Job.
+
+![Job Template form](/plugins/foreman_remote_execution/{{page.version}}/job_template_form.png)
+
+Before we get to inputs, lets look at other fields in Job tab. First
+one is autocomplete field **Job name**. This is generic name of the
+Job, which might be implemented by more Job Templates: for example one
+template can use use `yum` to install packages while second one uses
+`dnf`. **Job name** should be "install package" for both, while
+**Name** attributes would be something like "install package using
+yum" and "install package using dnf". This will become more handy when
+there are more than one **Provider type**. **Provider type** is
+another attribute which you don't have to change unless you install
+additional connection provider plugins. If you're unsure about the
+difference between **Name** and **Job name**, you can keep both the
+same, later you'll be using **Job name** to select which job you want
+to execute.
+
+On the very same tab you can configure inputs. These template inputs
+defines values that can be accessed in the template code by input
+macros based on it's name. To add new input, click green button "+ Add
+Input" which add more fields to the form. To remove the input, hit the
+cross in top right corner of the input form.
+
+We support four types of inputs. Each must have it's **name** which is
+used in macro. Let's say my input has name "motd", I can then use
+following ERB tag in my template code
+
+    echo "<%= input('motd') -%>" > /etc/motd
+
+When template is rendered for particular host, the `input('motd')` is
+evaluated like this:
+
+1. find and input with name motd
+2. determine its type
+3. based on type, fetch the value
+
+As you probably realized, the way how we determine the input value
+differes based on **Input type**. Let's start with User input. This
+input always need the value to be specified by user. It means when
+user runs the job, we display the field that he or she must fill in.
+Provided value will be used as a result for the corresponding input
+macro. Whatever you put in **Description** field will be displayed to
+the user so he knows what's the purpose of this input.
+
+Rest of types loads values from host and do not require any user
+interaction. Since the template is always rendered for specific host
+on which the job is executed we load the value this host data. These
+data can be either Fact (coming from Puppet/Salt/Chef), smart class
+parameter (when you use Foreman as Puppet/Salt ENC), smart variable or
```

Comment:
Maybe to be more generic instead naming individual configuration management tools? Ansible is coming up and this can get out of date :-)